charts/prow/templates/prow_configs.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "fullname" . }}
+data:
+  config.yaml: |-
+    ---
+    plank:
+      job_url_template: '{{ template "job_url_template" . }}'
+      report_template: '{{ template "report_template" . }}'
+      
+    triggers:
+    - repos:
+      {{ range .Values.repos }}
+      - {{ . }}
+      {{ end }}
+      trusted_org: {{ .Values.trustedOrg }}
+      
+    presubmits:
+      
+      {{ range .Values.repos }}
+      {{ . }}:
+      - name: "{{ . }}-pipeline"
+        always_run: true
+        context: "Jenkins GCE e2e"
+        rerun_command: "@prow go"
+        trigger: "@prow go"
+      {{ end }}
```

Comment:
(probably my lack of understanding of Prow) - what does this presubmit hook actually do? It doesn't seem to actually trigger something to run? I'm half way through deploying this chart myself, so I guess I'll find out soon 😄